modules/ROOT/pages/platform/create-account.adoc

```diff
@@ -1,14 +1,24 @@
 [[aura-create-account]]
 = Create an account
-:description: This page describes how to create a Neo4j Aura account.
+:description: This page describes how to create a Neo4j Aura account and a new instance.
 
-To access Neo4j Aura, you need to have an Aura account.
+== Sign up for Aura
+. Navigate to the Neo4j Aura Console.
+. Choose a sign-up method: Enter your email address and create a password, then select Register.
+Or, choose Continue with Google to sign up using your Google account.
+. You'll be presented with the Terms of Service and Privacy Policy. 
 
-To create an Aura account:
+== Access the new Aura console 
+. Once registered, you can use the Aura console.
+. To use the latest version of the console, click the banner at the top that says "Early access: check out the new Aura console. Preview now."
+
+== How to create an instance in Aura
+. To create a graph, you'll need to create an instance. 
+In the Aura console, click the *new instance* button.
+. Select your plan: Choose the Free instance option to start learning and experimenting. 
+You can upgrade to a paid plan later if you need more features or larger graphs.
+. Save your credentials: Copy or download the instance’s credentials (password, etc.) as a .txt file. 
+This will be required to access your instance later.
```

Comment:
The first step is to select the plan that suits your needs best.
You can upgrade your plan later if your needs change.
Copy or download your credentials as these are required to access your instance later. 
The password **is not available after this step and cannot be recovered**, so ensure you store it in a safe place.